### PR TITLE
Add FAQ to the docs.

### PR DESCRIPTION
**Pull request by MCGallaspy**
Does what it says. FAQ from [here](https://learningequality.org/ka-lite/faq/).

### CODE REVIEW

File: sphinx-docs/faq.rst [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+How do I install KA Lite?
+-------------------------
+
+Information on how to install KA Lite is available via our :doc:`user guides </usermanual/userman_main>`.
+
+How much does KA Lite cost to install?
+--------------------------------------
+
+It is **FREE**.
```
Comment (aronasorman) [inline sphinx-docs/faq.rst]: Might want to add an additional short sentence about us being free (gratis -- 0 USD) and free (open source, free to modify etc.) and that most of our content is open-licensed too.

**Comment by aronasorman** [comment]
Just one suggestion. Once that's fixed, I'll leave it to you @MCGallaspy to merge.

**Comment by mjptak** [comment]
perhaps a hook like this as well.   (from the mothership site.

**Comment by MCGallaspy** [comment]
@mjptak not clear what you mean... should I add an item to the FAQ, or are you suggesting the hook goes on all the pages in the docs?

**Comment by mjptak** [comment]
perhaps both but I think that little piece in the code has been pretty
effective for when I have tried to turn folk (with skills) onto Khan
Academy.  (gets back to the  "when they hit view source button we have
already won (:))".  Keep rockin it.

File: sphinx-docs/faq.rst [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+How do I install KA Lite?
+-------------------------
+
+Information on how to install KA Lite is available via our :doc:`user guides </usermanual/userman_main>`.
+
+How much does KA Lite cost to install?
+--------------------------------------
+
+It is **FREE** -- both free as in "free speech" and free as in "free beer"!
```
Comment (aronasorman) [inline sphinx-docs/faq.rst]: Can you add this link too? http://www.fsf.org/about/what-is-free-software

**Pull request merged**